src/pages/candidates/Candidates.jsx

```diff
@@ -26,15 +29,16 @@ const Candidates = () => {
   const [gridApi, setGridApi] = useState();
   const [anchorEl, setAnchorEl] = useState();
   const [isSendButtonDisabled, setIsSendButtonDisabled] = useState(true);
-  const [isAddToWorkButtonDisabled, setIsAddToWorkButtonDisabled] = useState(true);
+  const [isAddToWorkButtonDisabled, setIsAddToWorkButtonDisabled] =
+    useState(true);
   const open = !!anchorEl;
   const { id } = useParams();
 
   const listOfCandidates = useSelector((state) => state.candidates.candidates);
 
   const dispatch = useDispatch();
   const requestBody = {
-    pageSize: 20,
+    pageSize: 50,
```

Comment:
why is page size static if you can configure it in the table

Reply:
ok